### ISSUE TITLE
DllNotFoundException: Unable to load DLL 'native-googlesignin'

### ISSUE THREAD

**Issue by rossojr92** [googlesamples/google-signin-unity#267]
Hi
We started using this plugin long time ago.
Now, after updating Unity at 2022.3.50f1, on Android we are receiving this error log when we try to interact with google:

DllNotFoundException: Unable to load DLL 'native-googlesignin'. Tried the load the following dynamic libraries: Unable to load dynamic library 'native-googlesignin' because of 'Failed to open the requested dynamic library (0x06000000) dlerror() = dlopen failed: library "native-googlesignin" not found

Do you know any possible root cause for this?
Other relevant plugin in the project:
- GoogleMobileAds 9.3.0
- LevelPlay 8.5.0
- Firebase 11.9.0
- Facebook 17.0.0

I'm not sure where to start looking for the issue.

**Comment by mtribe-elaine** [comment]
Same issue, did you manage to solve it ?

**Comment by rossojr92** [comment]
Yes, in my case it was related to the External Dependency Manager for Unity.
The latest version of that plugin was creating a wrong entry in the settingsTemplate.gradle file.

It was generating:

`maven {
            url (unityProjectPath + "/Assets/GoogleSignIn/Editor/m2repository")
}`

intead of

`maven {
            url (unityProjectPath + "/Assets/GeneratedLocalRepo/GoogleSignIn/Editor/m2repository")
}`

Don't know why the new version of EDM4U is doing that. I had to fix it manually in the settingsTemplate.

**Comment by fzfking** [comment]
> Yes, in my case it was related to the External Dependency Manager for Unity. The latest version of that plugin was creating a wrong entry in the settingsTemplate.gradle file.
> 
> It was generating:
> 
> `maven { url (unityProjectPath + "/Assets/GoogleSignIn/Editor/m2repository") }`
> 
> intead of
> 
> `maven { url (unityProjectPath + "/Assets/GeneratedLocalRepo/GoogleSignIn/Editor/m2repository") }`
> 
> Don't know why the new version of EDM4U is doing that. I had to fix it manually in the settingsTemplate.

thanks, was same issue, helped

**Comment by usama-469** [comment]
Thank you soooo much, it helped!

**Comment by santhamethran** [comment]
@rossojr92 , @usama-469 
 
1. maven { url (unityProjectPath + "/Assets/GoogleSignIn/Editor/m2repository") }
2. maven { url (unityProjectPath + "/Assets/GeneratedLocalRepo/GoogleSignIn/Editor/m2repository") }

which is the correct one? i tried second one, it is not working me. 

<img width="1413" alt="Image" src="https://github.com/user-attachments/assets/01a578bd-7aaf-49f3-8ede-7da7faafd0cc" />


<img width="1413" alt="Image" src="https://github.com/user-attachments/assets/bd1609c8-dc71-4dd6-b453-dd2b4d7a3e62" />


<img width="1413" alt="Image" src="https://github.com/user-attachments/assets/db76513d-e620-47bf-b8e7-e4dac8a3286d" />


<img width="1413" alt="Image" src="https://github.com/user-attachments/assets/6087a25c-76a3-4b51-8b88-0c25bc5c6961" />
